### PR TITLE
Add admin seeder using Command Line Runner

### PR DESCRIPTION
**Pull request by Gabriel-Rockson**
Added command line runner to seed database with and admin on startup

### CODE REVIEW

**Review by daniel-bontii — CHANGES_REQUESTED** [review]
All your exceptions are the same. Think of combining them into one (my suggestions: ResourceAlreadyExistsException or EntityAlreadyExistsException) 
But feel free to come up with yours

File: pom.xml [hunk 1]
```diff
@@ -37,6 +37,12 @@
             <version>3.1.1</version>
         </dependency>
 
+        <!-- https://mvnrepository.com/artifact/io.github.cdimascio/dotenv-java -->
+        <dependency>
```
Comment (daniel-bontii) [inline pom.xml:41]: Spring boot provides a way to access the environment. This isn't bad but you could do it without an external dependency

File: src/main/java/com/amalitech/mentorshiptrackr/exceptions/PermissionAlreadyExistsException.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package com.amalitech.mentorshiptrackr.exceptions;
+
+public class PermissionAlreadyExistsException extends Exception {
```
Comment (daniel-bontii) [inline src/main/java/com/amalitech/mentorshiptrackr/exceptions/PermissionAlreadyExistsException.java:3]: Extend RuntimeException instead

Comment (Gabriel-Rockson) [reply src/main/java/com/amalitech/mentorshiptrackr/exceptions/PermissionAlreadyExistsException.java]: Alright

Comment (Gabriel-Rockson) [reply src/main/java/com/amalitech/mentorshiptrackr/exceptions/PermissionAlreadyExistsException.java]: Is there any Advantage to using that?

**Review by daniel-bontii — APPROVED** [review]

**Pull request merged**